psi4/driver/driver.py

```diff
@@ -736,6 +750,7 @@ def gradient(name, **kwargs):
         wfn = _process_displacement(energy, lowername, molecule, findif_meta_dict["reference"], 1, ndisp,
                                     **kwargs)
         var_dict = core.variables()
+        kwargs.pop('restart_file', None)
```

Comment:
Can I suggest a comment for posterity, about why we're popping `restart_file`? The same belongs in `hessian`.

Reply:
I added `# ensure displacement calculations do not use restart_file orbitals.`